### DISCUSSION TITLE
How to run this repo?

### DISCUSSION THREAD

**Discussion by vihsonic** [abharun/mini-chain#15] [Q&A] [answered]
I see `main` function in `src/bin/application.rs` file.
Please share TL:DR of this repo so we can easily onboard.

**Comment by abharun** [comment]
For rust project, the common entry file is `src/main.rs`

But regarding to project managment, `src/bin/application.rs` file is also the entry file. 
And you should add your own libraries in `src/lib.rs`.

**Reply by vihsonic** [reply]
what do you mean by my own libraries that should be imported in `sr/lib.rs` ?

**Reply by abharun** [reply, marked as answer]
just `.rs` files for our case.
including `client.rs`, `simulation.rs`, etc...

**Comment by abharun** [comment]
And the doc would be prepared for today evening.
